src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaCalculator, Draw {
+    private int radius;
+
+    public Circle() {
+
+    }
+
+    public Circle(int radius) {
+        this.radius = radius;
+    }
```

Comment:
```suggestion
    public Circle(String color, int radius) {
        super(color);
        this.radius = radius;
    }
```

let's create and use constructor of `Figure` class